runtime/gc_glue_java/ScavengerBackOutScanner.cpp

```diff
@@ -61,14 +61,18 @@ MM_ScavengerBackOutScanner::scanAllSlots(MM_EnvironmentBase *env)
 	MM_RootScanner::scanAllSlots(env);
 
 	if (!_extensions->isConcurrentScavengerEnabled()) {
-	/* Back out Ownable Synchronizer Processing */
+	/* Back out Ownable Synchronizer and continuation Processing */
 		MM_HeapRegionDescriptorStandard *region = NULL;
 		GC_HeapRegionIteratorStandard regionIterator(_extensions->heapRegionManager);
 		while (NULL != (region = regionIterator.nextRegion())) {
-			MM_HeapRegionDescriptorStandardExtension *regionExtension = MM_ConfigurationDelegate::getHeapRegionDescriptorStandardExtension(env, region);
-			for (uintptr_t i = 0; i < regionExtension->_maxListIndex; i++) {
-				MM_OwnableSynchronizerObjectList *list = &regionExtension->_ownableSynchronizerObjectLists[i];
-				list->backoutList();
+			if ((MEMORY_TYPE_NEW == (region->getTypeFlags() & MEMORY_TYPE_NEW))) {
+				MM_HeapRegionDescriptorStandardExtension *regionExtension = MM_ConfigurationDelegate::getHeapRegionDescriptorStandardExtension(env, region);
+				for (uintptr_t i = 0; i < regionExtension->_maxListIndex; i++) {
+					MM_OwnableSynchronizerObjectList *list = &regionExtension->_ownableSynchronizerObjectLists[i];
+					list->backoutList();
+					MM_ContinuationObjectList *continuationList = &regionExtension->_continuationObjectLists[i];
+					continuationList->backoutList();
+				}
```

Comment:
So, just to mention, Tenure Ownable Synchronizer lists are not handled any more. This is the change of behaviour outside Virtual Threads scope.

Reply:
you can merge the lines where you get the list with the line of calling backout. 
it shouldl be shorter than the line where you get the list alone due the class name being longer then backoutList method name

Reply:
add some more  comments:
for ownable synchronizers:
Tenure list might have gotten new elements during main Scavenge phase and it is important to restore the list, since it will be later iterated later during Marking Clearable phase.

for continuation:
Tenure cannot get new elements, since they are added only during Scavenge Clearable phase that cannot be reached if there was a Scavenge abort.